eox_tenant/settings/aws.py

```diff
@@ -43,3 +50,5 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
             'eox_tenant.middleware.AvailableScreenMiddleware',
             'eox_tenant.middleware.MicrositeCrossBrandingFilterMiddleware',
         ]
+
+        settings.AUTHENTICATION_BACKENDS = [EOX_TENANT_AUTH_BACKEND if (backend == EDX_AUTH_BACKEND) else backend for backend in settings.AUTHENTICATION_BACKENDS]  # pylint: disable=line-too-long
```

Comment:
@jfavellar90 Not sure if there is a more clean way to do this.

Reply:
@morenol I think it will work OK. And I agree with the fact of not appending the backend. Eox-tenant should only force the overwriting of the default auth backend, which is the one defined in EDX_AUTH_BACKEND